### PR TITLE
Seller profile

### PR DESCRIPTION
**Pull request by skripchenkoVeta**

### CODE REVIEW

File: app/controllers/products_controller.rb [hunk 1]
```diff
@@ -1,38 +1,75 @@
+# frozen_string_literal: true
+
 class ProductsController < ApplicationController
-    def index
-        @product = Product.all
+  def index
+    @products = Product.all.page(params[:page]).per(19)
```
Comment (Kaddyluck) [inline app/controllers/products_controller.rb:5]: Don't hardcode values, use CONSTANTS with proper names for that. What is 19? Why is 19?

File: app/controllers/products_controller.rb [hunk 2]
```diff
@@ -1,38 +1,75 @@
...
+  end
+
+  def menu
+    @salon = current_user.profileable.salons
+    @products = Product.where(salon_id: @salon)
```
Comment (Kaddyluck) [inline app/controllers/products_controller.rb:10]: Is it work? `@salon` - it's not id, it's just the list of user's salons

File: app/controllers/products_controller.rb [hunk 3]
```diff
@@ -1,38 +1,75 @@
...
+  end
+
+  def new
+    @product = Product.new
+  end
+
+  def create
+    @product = Product.create(product_params)
+
+    binding.pry
```
Comment (Kaddyluck) [inline app/controllers/products_controller.rb:24]: delete

File: app/controllers/salons_controller.rb [hunk 4]
```diff
@@ -1,38 +1,78 @@
+# frozen_string_literal: true
+
 class SalonsController < ApplicationController
-    def index
-        @salon = Salon.all
+  def index
+    @salons = Salon.all.page(params[:page]).per(19)
```
Comment (Kaddyluck) [inline app/controllers/salons_controller.rb:5]: 19 to constant

**Comment by skripchenkoVeta** [comment]
Thanks a lot for checking!

**Review by Kaddyluck — APPROVED** [review]

**Pull request merged**